### PR TITLE
DOCSUP-5854: Documented the SHOW SETTINGS statement

### PR DESCRIPTION
**Pull request by olgarev**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #18056

### CODE REVIEW

File: docs/_description_templates/template-engine.md [hunk 1]
```diff
@@ -58,6 +58,6 @@ Result:
 
 Follow up with any text to clarify the example.
 
-## See Also {#see-also}
+**See Also** {#see-also}
```
Comment (adevyatova) [inline docs/_description_templates/template-engine.md:61]:
```suggestion
**See Also** 
```

File: docs/_description_templates/template-function.md [hunk 2]
```diff
@@ -14,18 +14,16 @@ More text (Optional).
...
 
--   `x` — Description. [Type name](relative/path/to/type/dscr.md#type).
--   `y` — Description. [Type name](relative/path/to/type/dscr.md#type).
+-   `x` — Description. Optional (only for optional arguments). Possible values: <values list>. Default value: <value>. [Type name](relative/path/to/type/dscr.md#type). 
+-   `y` — Description. Optional (only for optional arguments). Possible values: <values list>.Default value: <value>. [Type name](relative/path/to/type/dscr.md#type). 
 
 **Parameters** (Optional, only for parametric aggregate functions)
 
--   `z` — Description. [Type name](relative/path/to/type/dscr.md#type).
+-   `z` — Description. Optional (only for optional parameters). Possible values: <values list>.Default value: <value>. [Type name](relative/path/to/type/dscr.md#type).
```
Comment (adevyatova) [inline docs/_description_templates/template-function.md:22]:
```suggestion
-   `z` — Description. Optional (only for optional parameters). Possible values: <values list>. Default value: <value>. [Type name](relative/path/to/type/dscr.md#type).
```

File: docs/en/sql-reference/statements/show.md [hunk 3]
```diff
@@ -428,4 +428,71 @@ errors_count:            0
...
+
+**Syntax**
+
+```sql
+SHOW [CHANGED] SETTINGS LIKE|ILIKE <name>
+```
+
+**Clauses**
+
+`LIKE` clause needs a setting name or part of the name without globs.
```
Comment (alesapin) [inline docs/en/sql-reference/statements/show.md:443]: The only difference between like and `LIKE` and `ILIKE` is case insensitiveness (ILIKE == case insensitive like). So `%` and `_` are also allowed. And I think we should give any details about `LIKE|ILIKE` hire, just something common: 
"`LIKE|ILIKE` allows to specify a matching pattern for setting name".

File: docs/_description_templates/template-function.md [hunk 4]
```diff
@@ -14,12 +14,12 @@ More text (Optional).
 
 **Arguments** (Optional)
 
--   `x` — Description. [Type name](relative/path/to/type/dscr.md#type).
--   `y` — Description. [Type name](relative/path/to/type/dscr.md#type).
+-   `x` — Description. Optional (only for optional arguments). Possible values: <values list>. Default value: <value>. [Type name](relative/path/to/type/dscr.md#type).
```
Comment (alesapin) [inline docs/_description_templates/template-function.md:17]: Not related to this PR?

**Review by alesapin — CHANGES_REQUESTED** [review]

File: docs/_description_templates/template-function.md (hunk 4 above)
Comment (olgarev) [reply docs/_description_templates/template-function.md:17]: No, it is not. Just a small fix.

**Review by alesapin — APPROVED** [review]

**Pull request merged**